metpy/plots/tests/test_mapping.py

```diff
@@ -98,7 +98,8 @@ def test_lcc_single_std_parallel():
 def test_mercator():
     """Test handling a mercator projection."""
     attrs = {'grid_mapping_name': 'mercator', 'standard_parallel': 25,
-             'longitude_of_projection_origin': -100, 'false_easting': 0, 'false_westing': 0}
+             'longitude_of_projection_origin': -100, 'false_easting': 0, 'false_westing': 0,
+             'central_longitude': 0}
```

Comment:
`central_latitude`. Probably need to update the commit message.

Reply:
Clearly a case of the Thursdays here

Reply:
You only have this once a week? This is just a day for me.